algebra-utils/src/msm/variable_base.rs

```diff
@@ -295,57 +544,108 @@ mod test {
         acc
     }
 
```

Comment:
Instead of duplicating the code for test_all_variants_<curve>(), if we are able to make a test_all_variants<G: AffineCurve> generic function and call it for each of the curve you want to test (into separate tests) it would be great.
See issue #76